frontend/src/components/Main/App.container.react.js

```diff
@@ -1,36 +1,38 @@
 // modules
-import { connect } from 'react-redux'
+import { connect } from "react-redux";
 // components
-import App from './App.presentational.react'
+import App from "./App.presentational.react";
 // views
-import { tabIndexView, numbersObjectView } from './App.reducer'
-import { expandModeView } from '../components/AppBar/AppBar.reducer'
+import { tabIndexView, numbersObjectView } from "./App.reducer";
+import { expandModeView } from "../components/AppBar/AppBar.reducer";
 // actions
 import {
   dispatchChangeTab,
   dispatchSetApi,
-  dispatchFetchInitialData,
-} from './App.action'
-import { dispatchLoadMore } from '../components/List/main/List.action'
+  dispatchFetchInitialData
+} from "./App.action";
+import { dispatchLoadMore } from "../components/List/main/List.action";
 // selectors
-import { getNumberOfTasksInEachLevel, getSortedTasks } from './App.selector'
+import { getNumberOfTasksInEachLevel, getSortedTasks } from "./App.selector";
+import { tasksView } from "./App.reducer";
 
 const mapStateToProps = state => ({
-  tasks: getSortedTasks(state),
+  // tasks: getSortedTasks(state),
```

Comment:
why?

Reply:
because there is no deadline in an added task this selector is enable to sort and has a function (i think reminded time ) that causes year 1970 to be added as default value of deadline and causes some problems